src/devices/auriol_4ld5661.c

```diff
@@ -45,10 +47,9 @@ static int auriol_4ld5661_decode(r_device *decoder, bitbuffer_t *bitbuffer)
 
         uint8_t *b  = bitbuffer->bb[i];
         int id      = b[0];
-        int battery  = b[1] & 0x80;
-        int channel  = ((b[1] & 0x30) >> 4) + 1;
+        int batt_ok = b[1] >> 7;
 
-        if (b[3] != 0xf0) {
+        if (b[3] != 0xf0 || (b[1] & 0x40) != 0) {
```

Comment:
This will ignore the two bits. But don't we want to check that it's 0 or 3? I.e. 1 or 2 are invalid.

Reply:
Depends, no idea if there are other brands of this rain meter out there that would use either ch1 or ch2 :/